documentation/element-api/tutorial-properties-attributes.asciidoc

```diff
@@ -115,18 +143,19 @@ element.getStyle().remove("background-color");
 element.getStyle().has("cursor");
 ----
 
-[NOTE]
-You cannot set the element's _style_ attribute using the `element.setAttribute` method.
-However you can use `element.getAttribute('style')` to get all the style rules in a single concatenated string.
+Note:
 
-[[textContent]]
-== TextContent
+* You cannot set an element's `style` attribute using the `element.setAttribute` method. 
+* You can use the `element.getAttribute('style')` method to get all style rules in a single concatenated string.
```

Comment:
As we remove the above, this and the whole Note section becomes obsolete. So let's remove this line and the above "Note:" too

Reply:
removed